### PR TITLE
docs: third-party licenses, TEST_COLORS, system()

### PR DESCRIPTION
**Pull request by justinmk**

### CODE REVIEW

File: test/busted/outputHandlers/nvim.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@ local global_helpers = require('test.helpers')
 
 -- Colors are disabled by default. #15610
 local colors = setmetatable({}, {__index = function() return function(s) return s end end})
-if os.getenv "NVIM_COLORS" then
+if os.getenv "TEST_COLORS" then
```
Comment (justinmk) [inline test/busted/outputHandlers/nvim.lua:6]: cc  @bfredl

File: runtime/doc/lua.txt [hunk 2]
```diff
@@ -301,7 +301,9 @@ arguments separated by " " (space) instead of "\t" (tab).
 							*:luafile*
 :[range]luafile {file}
 			Execute Lua script in {file}.
-			The whole argument is used as a single file name.
+                        The whole argument is used as the filename (like
+                        |:edit|), spaces do not need to be escaped.
+                        Alternatively you can |:source| Lua files.
```
Comment (gpanders) [inline runtime/doc/lua.txt:306]: Probably should stick with tabs here to stay consistent.

Comment (justinmk) [reply runtime/doc/lua.txt:306]: 95% of the file uses spaces so i converted the whole file

File: runtime/doc/repeat.txt [hunk 3]
```diff
@@ -172,15 +172,16 @@ Using Vim scripts					*using-scripts*
 For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 
 					*:so* *:source* *load-vim-script*
-:so[urce] {file}	Runs |Ex| commands or Lua code (".lua" files) read
-			from {file}.
+:[range]so[urce] {file}	Runs |Ex| commands or Lua code (".lua" files) from
+			{file}, or from the current buffer if no {file} is
+			given.
```
Comment (seandewar) [inline runtime/doc/repeat.txt:177]:
```suggestion
:[range]so[urce] [file]	Runs |Ex| commands or Lua code (".lua" files) from
			[file], or from the current buffer if no [file] is
			given.
```

**Pull request merged**